CrewChiefV4/Events/Fuel.cs

```diff
@@ -930,7 +941,51 @@ public void reportFuelStatus(Boolean allowNoDataMessage)
 
             Boolean reportedRemaining = reportFuelRemaining(allowNoDataMessage);
             Boolean reportedConsumption = reportFuelConsumption();
-            if (!reportedConsumption && !reportedRemaining && allowNoDataMessage)
+            Boolean reportedLitresNeeded = false;
+            int litresToEnd = getLitresToEndOfRace();
```

Comment:
This looks good, but does it trigger when "tell me more" or "verbose messages" is checked, or does it trigger all the time?

Reply:
Good point, this is a good candidate for the verbose stuff

Reply:
on reflection i don't think this is actually a good candidate for the tell-me-more stuff because it's primary use-case is when you get the information from a button press